docs/docs/reference/project-files/dashboards.md

```diff
@@ -6,6 +6,12 @@ sidebar_position: 30
 
 In your Rill project directory, create a `<dashboard_name>.yaml` file in the `dashboards` directory. Rill will ingest the dashboard definition next time you run `rill start`.
 
+:::info Did you know?
+
+Files that are *nested at any level* under your `dashboards` directory will be assumed to be metric definitions.
```

Comment:
I think either this is not working yet or there is a bug. I couldn't get it working.
cc : @ericpgreen2

Reply:
Correct, we haven't yet added support for a flexible project structure in Rill Developer. The backend reconciliation should work, but the frontend doesn't yet account for it.

Reply:
Okay cool I preempted this part then because it resulted in a [customer impacting issue](https://rilldata.slack.com/archives/C05693FLY7Q/p1700495656176499) (they had archived legacy models in a different subfolder but didn't include the "kind" -> resulting in backend error).

I think we can just wait to merge these changes once the frontend support is added